lambdas/shared/t4_lambda_shared/preview.py

```diff
@@ -55,6 +59,50 @@ def decompress_stream(chunk_iterator, compression):
             break
 
 
+def extract_fcs(file_, as_html=True):
+    """
+    parse and extract key metadata from parquet files
+
+    Args:
+        file_ - file-like object opened in binary mode (+b)
+
+    Returns:
+        dict
+            body - summary of main contents (if applicable)
+            info - metdata for user consumption
+    """
+    meta = {}
+    data = None
+    body = ""
+    info = {}
+    # per Lambda docs we can use tmp/*, OK to overwrite
+    file_path  = os.path.join(TEMP_DIR, "quilt-temp.fcs")
+    # fcsparser only takes paths, so we need to write to disk; OK because
+    # FCS files typically ~1MB
```

Comment:
This seems like a good tradeoff to get started, but FCS files can supposedly be large (e.g., the spec was updated to support sizes > 100MB).

Reply:
We have 500Mb of disk so we should be fine